### PR TITLE
Added ppc64le build script for emicklei/go-restful package

### PR DESCRIPTION
**Pull request by kumarvikas2020**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ? 

Note:
The Travis job will fail due to some test cases failing in parity with intel.
```# Observed 1 test failure for the version v3.7.3, which is in parity with Intel.
# FAIL    github.com/emicklei/go-restful/v3/examples/msgpack [build failed]```

### CODE REVIEW

File: g/go-restful/go-restful_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+if [ -z "$1" ]; then
+  export VERSION=v3.7.3
+else
+  export VERSION=$1
+fi
+if [ -d "goproxy" ] ; then
+  rm -rf goproxy
+fi
+
+dnf install -y git gcc
```
Comment (seth-priya) [inline g/go-restful/go-restful_ubi_8.4.sh:29]: is gcc needed?

File: g/go-restful/go-restful_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+else
+  go test -v ./...
+  ret=$?
+  if [ $ret -ne 0 ] ; then
+    echo "Tests failed "
+  else
+    echo "Build & unit tests Successful "
+  fi
+fi
+# Observed 1 test failure for the version v3.7.3, which is in parity with Intel.
```
Comment (seth-priya) [inline g/go-restful/go-restful_ubi_8.4.sh:71]: Please mention the names of the tests that are failing for future reference

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: g/go-restful/go-restful_ubi_8.4.sh (hunk 1 above)
Comment (kumarvikas2020) [reply g/go-restful/go-restful_ubi_8.4.sh:32]: Yes it is needed to run test case.

File: g/go-restful/go-restful_ubi_8.4.sh (hunk 2 above)
Comment (kumarvikas2020) [reply g/go-restful/go-restful_ubi_8.4.sh]: Added the name of failing test case.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**